webui/react/src/pages/TrialDetails/TrialDetailsWorkloads.tsx

```diff
@@ -233,13 +233,9 @@ const TrialDetailsWorkloads: React.FC<Props> = ({
     </ResponsiveFilters>
   );
 
-  // cleanup
   useEffect(() => {
     return () => {
       stopPolling();
```

Comment:
I wonder why some `stopPolling` are kept and others are removed

Reply:
cuz other files uses `stopPolling` somewhere else in the file